### PR TITLE
Add '/trivia move' command to trivia plugin

### PR DESCRIPTION
**Pull request by CameronClarry**

### CODE REVIEW

File: server/chat-plugins/trivia.js [hunk 1]
```diff
@@ -1703,6 +1703,58 @@ const commands = {
 	},
 	deletehelp: [`/trivia delete [question] - Delete a question from the trivia database. Requires: % @ # & ~`],
 
+	move(target, room, user) {
+		if (room.roomid !== 'questionworkshop') return this.errorReply('This command can only be used in Question Workshop.');
```
Comment (Zarel) [inline server/chat-plugins/trivia.js:1707]:
```suggestion
		if (room.roomid !== 'questionworkshop') {
			return this.errorReply('This command can only be used in Question Workshop.');
		}
```

File: server/chat-plugins/trivia.js [hunk 2]
```diff
@@ -1703,6 +1703,58 @@ const commands = {
...
+					question.category = category;
+					questions.splice(findEndOfCategory(category, false), 0, question);
+					writeTriviaData();
+					this.modlog('TRIVIAQUESTION', null, `changed category for '${param[1].trim()}' to '${param[0]}'`);
+					return this.privateModAction(`(${user.name} changed question category to '${param[0]}' for '${param[1].trim()}' from the question database.)`);
+				}
+			}
+		}
+	},
+	movehelp: [`/trivia move [category] | [question] - Change the category of question in the trivia databse. Requires: % @ # & ~`],
```
Comment (Zarel) [inline server/chat-plugins/trivia.js:1756]:
```suggestion
	movehelp: [
		`/trivia move [category] | [question] - Change the category of question in the trivia databse. Requires: % @ # & ~`
	],
```

**Comment by Kaiepi** [comment]
Looks OK to me as well.

File: server/chat-plugins/trivia.js [hunk 3]
```diff
@@ -1753,7 +1755,9 @@ const commands = {
 			}
 		}
 	},
-	movehelp: [`/trivia move [category] | [question] - Change the category of question in the trivia databse. Requires: % @ # & ~`],
+	movehelp: [
+		`/trivia move [category] | [question] - Change the category of question in the trivia databse. Requires: % @ # & ~`
```
Comment (Zarel) [inline server/chat-plugins/trivia.js:1759]:
```suggestion
		`/trivia move [category] | [question] - Change the category of question in the trivia databse. Requires: % @ # & ~`,
```

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**